example/train.py

```diff
@@ -0,0 +1,32 @@
+from keras.datasets import cifar10
+from keras.models import model_from_json
+import sys
+
+# Get the command line arguments
+model_file_name = ''
+try:
+    model_file_name = sys.argv[1]
+except IndexError:
+    print('Usage: python train.py model_json_file')
+    exit()
+
+# Load the dataset (keras.datasets.cifar10)
```

Comment:
You can add a comment on how users can upload their own datasets, or use other datasets from `keras.datasets`